src/lib/annotations/annotatorUtil.js

```diff
@@ -356,3 +356,26 @@ export function validateThreadParams(thread) {
     }
     return false;
 }
+
+/**
+ * Returns a function that passes a callback a location when given an event
+ *
+ * @param {Function} locationFunction - The function to get a location from an event
+ * @param {Function} callback - Callback to be called upon receiving an event
+ * @return {Function} Event listener to convert to document location
+ */
+export const eventToLocationHandler = (locationFunction, callback) => {
+    return (event) => {
+        if (event) {
+            if (event.stopPropagation) {
```

Comment:
I think we should just do a check on the event to see if it's either a TouchEvent, MouseEvent, or PointerEvent(some day we'll get this), and bail out if it isn't one of those, as the underlying functions used in here rely on a DOM Input event

OR

Assume it always has `stopPropagation()` and `preventDefault()`

Are there cases where those aren't present?